### PR TITLE
Fix func name PodResourcesLister

### PR DESCRIPTION
**Pull request by 70data**
The func PodResources is change to PodResourcesLister.
See this https://github.com/kubernetes/kubernetes/blob/release-1.14/pkg/kubelet/apis/podresources/v1alpha1/api.proto#L19
Please check.

### CODE REVIEW

**Comment by DanyC97** [comment]
hi @70data can you please sign the CLA?

**Comment by 70data** [comment]
I signed it.

**Comment by 70data** [comment]
@DanyC97 
Done.
Please check.

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ for these devices:
 ```gRPC
 // PodResources is a service provided by the kubelet that provides information about the
```
Comment (bradtopol) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: Why wouldnt you change PodResources to PodResourcesLister on this line as well?

**Comment by bradtopol** [comment]
Overall looking good. I had a question. See my inline comment

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md (hunk 1 above)
Comment (70data) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: Yep.
I forgot it.
I have fixed it.

**Comment by 70data** [comment]
@bradtopol 
Done.
Please check.

**Comment by bradtopol** [comment]
Great job!!!
/lgtm
/approve

**Pull request merged**